File: Python/Prime.py
def prime(n):
	if n==1:
		return False
	elif n==2:
		return True
	else:
		for i in range(2,n):
			if n%i==0:return False
			else:return True
n=int(input("Enter a number : "))
b=prime(n)
if(b==True):print("{0} is prime".format(n))
else:print("%d is not prime"%n)

File: C++/Prime.cpp
#include<iostream>
using namespace std;
class prime
{
    int n,i;
public:
    prime()
    {
        n=86;
    }
    void process()
    {
        for(i=2;i<=n/2;i++)
        {
            if(n%i==0)
            {
                cout<<n<<" is not prime number";
                break;
            }
        }
        if(i>n/2)
        {
            cout<<n<<" is prime number";
        }
    }
};
int main()
{
    prime p;
    p.process();
    return 0;
}
